CMakeLists.txt

```diff
@@ -678,8 +678,12 @@ else()
 endif()
 
 # Boost
-find_public_dependency(Boost REQUIRED COMPONENTS system date_time)
-target_link_libraries(torrent-rasterbar PUBLIC Boost::system Boost::date_time)
+find_public_dependency(Boost REQUIRED COMPONENTS system)
+target_link_libraries(torrent-rasterbar PUBLIC Boost::system)
+if (TORRENT_NO_DEPRECATE)
```

Comment:
This condition looks inverted. Also, where does this definition come from?
A brief look at this file, I can't see an option to disable deprecated functions in cmake.

Reply:
oh, I see. It's this one `deprecated-functions`. I was thrown off by the conditional around that option. I don't think the comment (or logic) there is right. the tests *can* be built without deprecated functions

Reply:
Can you tell me what exactly i need to put in `if ()`? I see no useful options to start from.

Reply:
The name is really confusing. I understand `NO_DEPRECATE` as "do not deprecate, allow old ABI".

Probably good name is `NO_DEPRECATED`.